src/modules/dashboard/components/ColonyHome/ColonyHomeLayout.css

```diff
@@ -30,12 +30,12 @@
 }
 
 .leftAside {
-  padding-top: 60px;
+  padding-top: 0px;
```

Comment:
I know its nitpick, but we always set 0 as 0 not "0px" so lets be consistent in here 🙈

Reply:
Yes sir!